### ISSUE TITLE
Record RDT Result Page Enhancement

### ISSUE THREAD

**Issue by ellyasafaa** [onaio/rdt-standard#804]
![klik tombol ini hingga berwarna hijau untuk memunculkan detail pertanyaan kontak (tracing kontak) (8)](https://user-images.githubusercontent.com/73918009/108788046-b2864980-75a9-11eb-84c0-5a5d4770c63c.png)

I think the automated RDT type feature has been fixed, right?

**Comment by vincent-karuri** [comment]
A lot of these questions have been addressed before in previous conversations, including here https://github.com/onaio/rdt-standard/issues/747.

**Comment by ellyasafaa** [comment]
@vincent-karuri Yes, I am aware of that. Just to point out the display to make it clearer and there is one issue that I think it is really important to have and hasn't been pointed out: the image captured appears on the result page.
